R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -1,43 +1,91 @@
+#' - `$to_arrow_record_batch_stream_reader()`
+#' - `$to_arrow_table()`
+#' - `$to_tibble()`
+#' - `$to_data_frame()`
+#'
+#' @examples
+#' 
+#' # connect to the Deephaven server running on "localhost:10000" using anonymous 'default' authentication
+#' client_options <- ClientOptions$new()
+#' client <- Client$new(target="localhost:10000", client_options=client_options)
+#' 
+#' # open a table that already exists on the server
+#' new_table_handle1 <- client$open_table("table_on_the_server")
+#' 
+#' # convert the Deephaven table to an R data frame
+#' new_data_frame <- new_table_handle1$to_data_frame()
+#' 
+#' # modify new dataframe in R
+#' new_data_frame$New_Int_Col <- c(1, 2, 3, 4, 5)
+#' new_data_frame$New_String_Col <- c("I", "am", "a", "string", "column")
+#' 
+#' # push new dataframe to the server and name it "new_table"
+#' new_table_handle2 <- client$import_table(new_data_frame)
+#' new_table_handle2$bind_to_variable("new_table")
 
 TableHandle <- R6Class("TableHandle",
     public = list(
 
         initialize = function(table_handle) {
-
             if (class(table_handle)[[1]] != "Rcpp_INTERNAL_TableHandle") {
                 stop("'table_handle' should be an internal Deephaven TableHandle. If you're seeing this,
                 you are trying to call the constructor of TableHandle directly, which is not advised.")
             }
             private$internal_table_handle <- table_handle
         },
 
+        #' @description
+        #' Binds the table referenced by this TableHandle to a variable on the server,
+        #' enabling it to be accessed by that name from any Deephaven API.
+        #' @param name New name for this table on the server.
         bind_to_variable = function(name) {
             if (class(name)[[1]] != "character") {
                 stop(paste("'name' should be a character or a string. Got object of type ", class(name)[[1]], " instead."))
             }
             private$internal_table_handle$bind_to_variable(name)
         },
 
+        #' @description
+        #' Imports the table referenced by this TableHandle into an Arrow RecordBatchStreamReader.
+        #' @return A RecordBatchStreamReader containing the data from the table referenced by this TableHandle.
         to_arrow_record_batch_stream_reader = function() {
             ptr = private$internal_table_handle$get_arrow_array_stream_ptr()
             rbsr = RecordBatchStreamReader$import_from_c(ptr)
             return(rbsr)
         },
 
+        #' @description
+        #' Imports the table referenced by this TableHandle into an Arrow Table.
+        #' @return A Table containing the data from the table referenced by this TableHandle.
         to_arrow_table = function() {
             rbsr = self$to_arrow_record_batch_stream_reader()
             arrow_tbl = rbsr$read_table()
             return(arrow_tbl)
         },
 
+        #' @description
+        #' Imports the table referenced by this TableHandle into a dplyr Tibble.
```

Comment:
dplyr Tibble?

Reply:
Yes, I updated all of the places mentioning tibble to be dplyr Tibble rather than R Tibble